### PR TITLE
Fix bug not allowing users to close by clicking on backdrop

### PR DESCRIPTION
**Pull request by guyathomas**
**Description of changes:**
It seems `!event.target.classList.contains(className('view')` had the intention of only allowing the closing of the backdrop on clicking in a selected region (picture below). However, this area did not actually take up the entire backdrop. Instead of whitelisting allowed areas, i decided to blacklist areas that a click should not close ( previous and next buttons )

image

**Related issues (if any):**
https://github.com/jossmac/react-images/issues/302

**Checks:**

- [x] Please confirm `yarn run lint` ran successfully
- [x] Please confirm that only `/src` and `/examples/src` are committed
- [ ] [if new feature] Please confirm that documentation was added to the README.md

### CODE REVIEW

File: src/components/Carousel.js [hunk 1]
```diff
@@ -209,11 +209,13 @@ class Carousel extends Component<CarouselProps, CarouselState> {
     }
   };
 
-  prev = () => {
+  prev = (event) => {
+    event.stopPropagation();
```
Comment (bigklubba) [inline src/components/Carousel.js]: this is needed for the close and fullscreen button as well

**Comment by kyriiherman** [comment]
hey, any updates on this PR? 🤔

**Comment by davwheat** [comment]
I'll need to test the changes and fix the linting error (unless the PR author does it), then it can be merged!

**Comment by davwheat** [comment]
Fixed by #357

**Pull request closed without merging**